lib/encoder.go

```diff
@@ -202,7 +210,7 @@ type simpleCertificate struct {
 	URINames              []string            `json:"uri_names,omitempty"`
 	EmailAddresses        []string            `json:"email_addresses,omitempty"`
 	SCTList               []*simpleSCT        `json:"sct_list,omitempty"`
-	Warnings              []string            `json:"warnings,omitempty"`
```

Comment:
We can't change the name of this public struct member just in case anyone is using it, so we'll have to keep it as `Warnings`. Changing the output of the tool like you did is fine, though.